snapcraft/__init__.py

```diff
@@ -222,6 +223,20 @@ def build(self):
             ignore=lambda d, s: common.SNAPCRAFT_FILES
             if d is self.sourcedir else [])
 
+    def clean_build(self):
```

Comment:
Yeah it's a bit unfortunate. But the function calls are `pull()`, `build()`, `stage()` and `strip()`, no *_step. So I feel like I need to match up the clean_* functions the same way.